src/main/kotlin/com/zepben/evolve/services/network/translator/NetworkCimToProto.kt

```diff
@@ -330,6 +333,17 @@ fun toPb(cim: TownDetail, pb: PBTownDetail.Builder): PBTownDetail.Builder =
         stateOrProvince = cim.stateOrProvince
     }
 
+fun toPb(cim: StreetDetail, pb: PBStreetDetail.Builder): PBStreetDetail.Builder =
```

Comment:
Can you please put the functions in each section in alphabetical order. It makes it much easier to spot if things have been missed later